astropy/visualization/wcsaxes/__init__.py

```diff
@@ -6,6 +6,7 @@
 try:
     import pytest
     pytest.importorskip("matplotlib")
+    pytest.importorskip("pytest-mpl")
```

Comment:
Shouldn't these live in the test files rather than in the module itself? I'm fine with the fix as it's just extends the current approach, but wonder whether the current approach is optimal or not.

Reply:
I was confused too....wondering how the `wcsaxes` tests are skipped if `mpl` isn't installed because they are written such that they *require* `mpl`.  They do not check if `mpl` is available, they just import `mpl`.  I discovered how the tests are skipped in this `__init__` file.  We could replace this by adding the `mpl` checks, but the entire `visualization.wcsaxes` subpackage *requires* `mpl`  @astrofrog wrote this, so I'll let him comment.